src/often.js

```diff
@@ -16,10 +16,10 @@ module.exports = async ({github, context, owner, repo, workflow}) => {
             repo,
             workflow_id
         });
-        console.log(``)
+
         let runObj = JSON.stringify(runs)
-        console.log(`Runs: ${runObj}`)
-        console.log(``)
+        console.log(`\nRuns: ${runObj}\n`)
```

Comment:
Ha, let's NOT do that :-p

```suggestion
        console.log(`Runs: ${runObj}`)
```